README.md

```diff
@@ -24,6 +24,14 @@ The following variables are available for "File Title Template":
 
 More variables going to be added on upcoming releases.
 
+## Setup/Configuration
+After the plugin has been installed into IntelliJ, go to menu
+
+   `File` --> `Settings` --> `Appearence` & `Behaviour` --> `Custom Title`
```

Comment:
Can you fix the typo in `Appearence`?

Reply:
moreover, I think this tree is valid for Windows and Linux users, so it would be awesome to add a similar guide for macOS users too: `IntelliJ IDEA` --> `Preferences...` --> `Appearance & Behaviour` --> `Custom Title`

Reply:
I've correctedf the typo